frontend/packages/kubevirt-plugin/src/components/vm-templates/table/VMTemplateTable.tsx

```diff
@@ -51,6 +53,11 @@ const vmTemplateTableHeader = (showNamespace: boolean, t: TFunction) =>
         sortFunc: 'vmTemplateProvider',
         transforms: [sortable],
       },
+      {
+        title: t('kubevirt-plugin~Support Level'),
```

Comment:
Some headers are sentence case (Support level) and others are title case (Support Level). We need to stay consistent with whatever console is using.

Reply:
since we have a mixture of sentence case and title case in the templates table headers, we will approach this on a different BZ